docs/src/components/AIToggle/index.tsx

```
@@ -0,0 +1,264 @@
+              <code className={styles.overlayUrl}>{mdUrl}</code>
+            </div>
+            <div className={styles.overlayActions}>
+              {markdown && (
+                <>
+                  <button
+                    type="button"
+                    className={styles.overlayBtn}
+                    onClick={handleCopy}
+                  >
+                    <svg width="12" height="12" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
+                      {copied ? (
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)